swxsoc/util/tests/test_util.py

```diff
@@ -1,9 +1,9 @@
 """Tests for util.py"""
 
+import importlib
 import pytest
-
 from astropy.time import Time
-from swxsoc.util import util
```

Comment:
Why do we need to do a lazy import of the util module here?

Reply:
I'm trying to figure out the best way to test out the configuration changes since the module is configured on import, and I haven't been able to figure out how to re-initialize it.

Reply:
I added a function `reconfigure` for testing to be able to do this